Tests/test_file_gbr.py

```diff
@@ -15,3 +15,10 @@ def test_gbr_file():
     with Image.open("Tests/images/gbr.gbr") as im:
         with Image.open("Tests/images/gbr.png") as target:
             assert_image_equal(target, im)
+
+
+def test_multiples_operation():
+    with Image.open("Tests/images/gbr.gbr") as im:
+        rect = (0, 0, 10, 10)
+        im.crop(rect)
+        im.crop(rect)
```

Comment:
```suggestion
        im.load()
        im.load()
```
Simplify the test case?

Reply:
As you want. I am not fine connoiseur of this library.

I used `crop` as an example instead of `load` because of the lazy loading (and because in my use case I do not call `load()` explicitly)

But since the fix is clearly on the the `load` method it would propably be clearer.

Reply:
Ok, sure. I've pushed some changes, including this, and also rearranged your change to the plugin itself to match some of our existing code -
https://github.com/python-pillow/Pillow/blob/bc1dc14b592dbc7d303bc7a82daa13fcdc547369/src/PIL/IcoImagePlugin.py#L291-L294